### PR TITLE
WD-10162 copy update

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Copy update

## QA

- Go to https://ubuntu-com-13748.demos.haus/desktop/developers and compare with [copy doc changes](https://docs.google.com/document/d/1VJ3xT2ViC-CvL0UmsAUzpYFmvpuqUP4AbpnlPm8tqOA/edit) (remove image and hyphens)
- Changes to charts are automated and already [in the branch](https://github.com/canonical/ubuntu.com/pull/13734)

- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10162

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/desktop/developers.html [hunk 1]
```diff
@@ -4,503 +4,540 @@
...
-      {{ image (
-        url="https://assets.ubuntu.com/v1/1819c545-woman-working.jpg",
-        alt="",
-        width="2464",
-        height="1052",
-        hi_def=True,
-        loading="auto"
-        ) | safe
-      }}
-    </div>
```
Comment (carkod) [inline templates/desktop/developers.html:38]: Removed image. One of the few changes. Most of the others are djlint formatting

File: templates/desktop/developers.html [hunk 2]
```diff
@@ -4,503 +4,540 @@
...
-    <div class="col">
-      <h2 class="p-text--small-caps u-no-margin--bottom">Why choose Ubuntu Desktop?</h2>
-      <p class="p-heading--2">The developers' choice</p>
-    </div>
-    <div class="col">
-      <ul class="p-list--divided u-no-margin--bottom">
-        <li class="p-list__item is-ticked">Using Ubuntu Desktop provides a common platform for development, test, and production environments.</li>
-        <li class="p-list__item is-ticked">Tools, such as <a href="https://juju.is/">Juju</a>, <a href="https://microk8s.io/">Microk8s</a>, and <a href="https://multipass.run/">Multipass</a> make developing, testing, and cross-building easy and affordable.</li>
-        <li class="p-list__item is-ticked"><strong>Long term support (LTS)</strong> releases are delivered every 2-years, with 5 years of standard support extended to 10 years with an Ubuntu Pro Desktop subscription.</li>
-        <li class="p-list__item is-ticked">New Developer editions of Ubuntu Desktop are released every 6-months, with 9-months of support and access to the latest kernel, libraries, and new upstream OS features.</li>
```
Comment (carkod) [inline templates/desktop/developers.html:54]: Removing the hyphen. Everything else is Djlint formatting

File: templates/desktop/developers.html [hunk 3]
```diff
@@ -4,108 +4,136 @@
...
         alt="",
-        width="2464",
-        height="1052",
+        width="2120",
+        height="906",
         hi_def=True,
         loading="auto"
         ) | safe
       }}
     </div>
```
Comment (carkod) [inline templates/desktop/developers.html:48]: Updated image, everything else on this file is djlint formatting.

File: templates/desktop/developers.html [hunk 4]
```diff
@@ -4,108 +4,136 @@
...
-  <div class="p-section">
-    <div class="u-fixed-width">
-      {{ image (
-        url="https://assets.ubuntu.com/v1/1819c545-woman-working.jpg",
-        alt="",
-        width="2464",
-        height="1052",
+    <div class="p-section u-fixed-width">
+      {{
+        image(
```
Comment (akbarkz) [inline templates/desktop/developers.html:39]: this bit doesn't seem to be formatted

File: templates/desktop/developers.html [hunk 5]
```diff
@@ -4,108 +4,136 @@
...
-          ) | safe
-          }}
-        </div>
-        <div class="p-logo-section__item">
-          {{ image (
+    <div class="u-fixed-width">
+      <div class="p-logo-section">
+        <div class="p-logo-section__items">
+          <div class="p-logo-section__item ">
+            {{ image (
```
Comment (akbarkz) [inline templates/desktop/developers.html:124]: please remove space between `image` and the bracket

File: templates/desktop/developers.html (hunk 4 above)
Comment (carkod) [reply templates/desktop/developers.html:39]: I formatted it by hand, isn't this better?

File: templates/desktop/developers.html [hunk 6]
```diff
@@ -4,108 +4,136 @@
...
-      {{ image (
-        url="https://assets.ubuntu.com/v1/1819c545-woman-working.jpg",
-        alt="",
-        width="2464",
-        height="1052",
+    <div class="p-section u-fixed-width">
+      {{
+        image(
+        url="https://assets.ubuntu.com/v1/bbe34b80-woman-working.jpg",
+        alt="Woman working",
```
Comment (juanruitina) [inline templates/desktop/developers.html:41]: Let's improve it a bit: "Developer sitting in front of her computer in an office"

**Review by akbarkz — APPROVED** [review]

File: templates/desktop/developers.html (hunk 4 above)
Comment (akbarkz) [reply templates/desktop/developers.html]: We should not be discussing what is better and what is not, we already have a linter for that, let it decide how it should look like

**Pull request merged**